lib/ui/semantics/semantics_node.h

```diff
@@ -142,6 +142,7 @@ struct SemanticsNode {
   std::vector<int32_t> childrenInTraversalOrder;
   std::vector<int32_t> childrenInHitTestOrder;
   std::vector<int32_t> customAccessibilityActions;
+  int32_t headingLevel = -1;
```

Comment:
why not use 0 as `not heading` instead of -1?

Reply:
Ok, done.